implementations/rust/ockam/ockam_api/src/nodes/service.rs

```diff
@@ -239,7 +240,9 @@ impl NodeManager {
                 .await?
                 .is_none()
             {
-                let expression = expression.unwrap_or(Expr::CONST_TRUE);
+                let check_credential_expression =
+                    Expr::Ident(format!("{}.{}", SUBJECT_KEY, ABAC_HAS_CREDENTIAL_KEY));
```

Comment:
So this is the "fallback" expression? Why is that? A comment with the context or the relation with [this code block](https://github.com/build-trust/ockam/pull/7513/files#diff-fe4b8061645846b4e6d042a8d8b682b21f8e7d70e50657f62ae0e2fdbb7a2513R120-R123) would be helpful.

Reply:
It is default policy and because I changed ABAC implementation, this
```
let check_credential_expression = Expr::Ident(format!("{}.{}", SUBJECT_KEY, ABAC_HAS_CREDENTIAL_KEY));
```
now behaves exactly like this
```
let expression = expression.unwrap_or(Expr::CONST_TRUE);
```
was behaving before.
Therefore, nothing really changed here